backend/README.md

```diff
@@ -8,9 +8,11 @@
 2. Create a file called `.env`. Learn about these files [here](https://medium.freecodecamp.org/nodejs-custom-env-files-in-your-apps-fa7b3e67abe1).
 3. Inside that file create a variable called `MONGODB_URI` and set it to `mongodb://localhost:27017/[localservername]` or the URI associated with your deployment on MongoDB Atlas, mLab or any other similar provider.
 4. This server uses [auth0](https://auth0.com/) for authentication, and [Twilio](https://www.twilio.com/) + [Sendgrid](https://sendgrid.com/) for text message/email alerts. To locally run these features you need to create an account on the respective providers and follow their instructions. Set four more environmental variables: `TWILIO_SID`, `TWILIO_TOKEN`, `TWILIO_NUMBER`, and `SENDGRID_API_KEY`.
-5. Install node modules `yarn`
-6. Run server `yarn server`
-7. Go to http://localhost:5000 the view the GraphQL documentation and http://localhost:5000/graphql to view GraphQL playground.
+5. Create variables for `STRIPE_SECRET_KEY` and `STRIPE_PUBLISHABLE_KEY` and use the same values used in the front end setup.
```

Comment:
As far as I'm aware, the stripe publishable key isn't needed for the backend. Also, the stripe secret key *definitely is not* to be used in the frontend.